src/components/KEmptyState/KEmptyState.vue

```diff
@@ -43,6 +51,9 @@
 <script lang="ts" setup>
 import KButton from '@/components/KButton/KButton.vue'
 import KIcon from '@/components/KIcon/KIcon.vue'
+import { useSlots } from 'vue'
+
+const slots = useSlots()
```

Comment:
nit: since you're only using `slots` in the template, you can remove the const declaration here along with the import form `vue` and access it as a dollar sign variable in the template like so: `v-if="$slots.title"`.

Reply:
This was changed now.